### PR TITLE
chore: change the log background to same as severity text with some opacity

### PR DESCRIPTION
**Pull request by vikrantgupta25**
### Summary

- remove the drawer mask when using logs details drawer 
- rather than overriding the severity indicator use the same color and make the entire line of similar color with opacity 10.

#### Related Issues / PR's

https://signoz-team.slack.com/archives/C02TJ466H8U/p1727762326163239?thread_ts=1727517688.741869&cid=C02TJ466H8U

#### Screenshots

https://github.com/user-attachments/assets/735fe1a1-e5a8-4410-9bc4-962b457568d0

#### Affected Areas and Manually Tested Areas

- logs details view 
- raw / list / column and context view. 

----

> [!IMPORTANT]
> Update log background color to match severity text with opacity adjustments across components and styles.
> 
>   - **Behavior**:
>     - Updated log background color to match severity text with opacity in `getActiveLogBackground()` in `utils/logs.ts`.
>     - Removed `isActive` prop from `LogStateIndicator` in `LogStateIndicator.tsx`.
>   - **Components**:
>     - Updated `LogStateIndicator` to remove `isActive` logic.
>     - Adjusted `Container` and `RawLogViewContainer` styles to use `getActiveLogBackground()`.
>   - **Styles**:
>     - Removed `isActive` styles from `LogStateIndicator.styles.scss`.
>     - Updated `TableRow.styles.scss` and `InfinityTableView/styles.ts` to use new background logic.
>   - **Misc**:
>     - Removed unused imports and props in `useTableView.tsx` and `InfinityTableView/index.tsx`.
> 
> <sup>This description was created by </sup>[Ellipsis](https://www.ellipsis.dev?ref=SigNoz%2Fsignoz&utm_source=github&utm_medium=referral)<sup> for dfdfccdbde249a0b156ed3fc1c46c5330fed19a7. It will automatically update as commits are pushed.</sup>

### CODE REVIEW

File: frontend/src/components/Logs/LogStateIndicator/LogStateIndicator.styles.scss [hunk 1]
```diff
@@ -41,10 +41,4 @@
 			background-color: var(--bg-sakura-500);
 		}
 	}
-
-	&.isActive {
-		.line {
```
Comment (vikrantgupta25) [inline frontend/src/components/Logs/LogStateIndicator/LogStateIndicator.styles.scss:46]: we do not need the log-state indicator to change the color anymore as we will be giving the log type indicator with some opacity in the background

File: frontend/src/components/Logs/LogStateIndicator/LogStateIndicator.tsx [hunk 2]
```diff
@@ -44,22 +44,16 @@ export const LogType = {
...
 	return (
-		<div className={cx('log-state-indicator', isActive ? 'isActive' : '')}>
+		<div className="log-state-indicator">
 			<div className={cx('line', type, fontSize)}> </div>
 		</div>
 	);
 }
 
-LogStateIndicator.defaultProps = {
-	isActive: false,
```
Comment (vikrantgupta25) [inline frontend/src/components/Logs/LogStateIndicator/LogStateIndicator.tsx:62]: not needed anymore, same reason as above.

**Comment by GeekBoySupreme** [comment]
This looks good 🙌🏼

File: frontend/src/components/Logs/ListLogView/index.tsx [hunk 3]
```diff
@@ -195,21 +195,20 @@ function ListLogView({
...
 			>
 				<div className="log-line">
-					<LogStateIndicator
-						type={logType}
-						isActive={
-							activeLog?.id === logData.id || activeContextLog?.id === logData.id
-						}
-						fontSize={fontSize}
-					/>
+					<LogStateIndicator type={logType} fontSize={fontSize} />
```
Comment (YounixM) [inline frontend/src/components/Logs/ListLogView/index.tsx:211]: We don't need to pass if the log is active ?

**Review by YounixM — APPROVED** [review]

**Pull request merged**